spec.html

```diff
@@ -31492,7 +31492,7 @@ <h1>The Array Constructor</h1>
         <li>creates and initializes a new Array exotic object when called as a constructor.</li>
         <li>also creates and initializes a new Array object when called as a function rather than as a constructor. Thus the function call `Array(&hellip;)` is equivalent to the object creation expression `new Array(&hellip;)` with the same arguments.</li>
         <li>is a function whose behaviour differs based upon the number and types of its arguments.</li>
-        <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the exotic Array behaviour must include a `super` call to the Array constructor to initialize subclass instances that are Array exotic objects. However, most of the `Array.prototype` methods are generic methods that are not dependent upon their *this* value being an Array exotic object.</li>
+        <li>may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the exotic Array behaviour must include a `super` call to the Array constructor to initialize subclass instances that are Array exotic objects. However, most of the `Array.prototype` methods are generic methods that are not dependent upon their *this* value being an Array exotic object.</li>
```

Comment:
The same, subclassing Array is really an important feature in ES6, why discourage that

Reply:
I disagree? I guess it depends on what you mean by subclassing, since in JS it encompasses stuff like species construction, and that methods can deal with non-Array objects that still have the Array exotic objects' slots and essential methods.

What about subclassing Arrays is important beyond being able to `extend` them?